app/tests/test_views.py

```diff
@@ -0,0 +1,11 @@
+from app.views import (
+    TranscribeSubmission,
+    accept_coc,
+    choose_transcription,
+    unclaim_submission,
+)
+
+asdf = TranscribeSubmission
+bsdf = accept_coc
+csdf = choose_transcription
+dsdf = unclaim_submission
```

Comment:
Peak variable naming :P

Reply:
I just needed it to stop yelling at me so that I could commit the real stuff lol. It's fixed now.